### ISSUE TITLE
bug: Correct minSdkVersion for platform target android

### ISSUE THREAD

**Issue by iamwolf** [driftyco/ionic#3777]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>2.x</span>  <span ionic-webview>webview</span>

<span ionic-description>When adding android as a platform target, the Android manifest has a default minSdkVersion=10 (2.3.3). ionic has dropped support for such old Android versions for a while now and leaving the standard minSdkVersion at 10 will result in ionic apps being shown in the Play store as compatible for old devices while, in fact, they are not.

Fix: set minSdkVersion 16 (Jelly Bean)

To reproduce:
```
ionic start test
cd test
ionic platform add android
cat platforms/android/AndroidManifest.xml
```
</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @iamwolf!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3777). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3777)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by perrygovier** [comment]
Fixed in 78cf108ea475ffbf7a454bff341ff5268d2dca7b
